02-peer-protocol.md

```diff
@@ -633,19 +673,20 @@ There are four values that need be derived:
 1. The `cltv_expiry_delta` for channels, `3R+2G+2S`: if in doubt, a
    `cltv_expiry_delta` of 12 is reasonable (R=2, G=1, S=2).
 
-2. For sent HTLCs: the timeout deadline after which the channel has to be failed
+2. The `cltv_expiry_delta` for sent HTLCs: the timeout deadline after which the channel has to be failed
    and timed out on-chain. This is `G` blocks after the HTLC's
    `cltv_expiry`: 1 block is reasonable.
 
-3. For received HTLCs (with a preimage): the fulfillment deadline after which
+3. The `cltv_expiry_delta` for received HTLCs (with a preimage): the fulfillment deadline after which
```

Comment:
Similarly:

3. The deadline for received HTLCs this node has fulfilled: the deadline after which...